runtime/gc_structs/VMThreadStackSlotIterator.cpp

```diff
@@ -87,31 +123,9 @@ GC_VMThreadStackSlotIterator::scanSlots(
 		)
 {
 	J9StackWalkState stackWalkState;
-	J9JavaVM *vm = vmThread->javaVM;
-
-	stackWalkState.objectSlotWalkFunction = vmThreadStackDoOSlotIterator;
-	stackWalkState.userData1 = (void *)oSlotIterator;
-	stackWalkState.userData2 = (void *)vm;
-	stackWalkState.userData3 = userData;
-
-	stackWalkState.flags = J9_STACKWALK_ITERATE_O_SLOTS | J9_STACKWALK_DO_NOT_SNIFF_AND_WHACK;
+	GC_VMThreadStackSlotIterator::initializeStackWalkState(&stackWalkState, vmThread, userData, oSlotIterator, includeStackFrameClassReferences, trackVisibleFrameDepth);
```

Comment:
Sorry for being picky and not clear enough, but we don't need the class scope operator here (and 2 other spots). My original concern was only about for the symbol definition (and potential overlap with a same symbol somewhere else).